.releaserc

```diff
@@ -0,0 +1,77 @@
+{
+  "branches": ["master"],
```

Comment:
It's worth creating a pre-release branch config - you'll be able to test release process this way without messing with the master version